### PR TITLE
remove triu from fluid

### PR DESCRIPTION
**Pull request by ccrrong**
### PR types

Others

### PR changes

APIs

### Describe

To clean the code in paddle.fluid, the useless codes should be removed and code still be used should move to a new position.

For function paddle.fluid.layers.tensor.triu, since it is not used in version 2.x, should be removed.

### CODE REVIEW

**Review by heavengate — DISMISSED** [review]

**Review by jzhang533 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/mlu/test_tril_triu_op_mlu.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def test_fluid_api(self):
             with program_guard(prog, startup_prog):
                 data = np.random.random([1, 9, 9, 4]).astype(dtype)
                 x = fluid.data(shape=[1, 9, -1, 4], dtype=dtype, name='x')
-                triu_out = fluid.layers.triu(x)
+                triu_out = tensor.triu(x)
```
Comment (XiaoguangHu01) [inline python/paddle/fluid/tests/unittests/mlu/test_tril_triu_op_mlu.py]: 请使用2.0推荐公开API调用方式，paddle.triu

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
LGTM

**Pull request merged**